code/modules/SCPmodular/scps/euclid/SCP-173.dm

```diff
@@ -0,0 +1,212 @@
+/mob/living/simple_animal/hostile/scp_173
+	name = "SCP-173"
+	desc = "A statue, constructed from concrete and rebar with traces of Krylon brand spray paint"
+	icon = 'icons/mob/scpicon/scpmobs/scp-173.dmi'
+	ventcrawler = VENTCRAWLER_NUDE
+	icon_state = "173"
+
+	maxHealth = 5000
+	health = 5000
+	move_force = MOVE_FORCE_NORMAL
+	move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	pull_force = MOVE_FORCE_EXTREMELY_STRONG
+	response_help = "touches"
+	response_disarm = "pushes"
+	a_intent = INTENT_HARM
+//	possible_a_intents = list(INTENT_HARM)
```

Comment:
Don't leave commented-out code.

Reply:
fixed in testing, next PR wont have this